proteus/tests/ci/test_poisson.py

```diff
@@ -103,7 +103,7 @@ def test_c0p2(self):
         del ns
         assert(True)
 
-    def check_c0q1(self,test_hexMesh_3x3=False,use_petsc=False,name="_hexMesh_"):
+    def check_c0q1(self,test_hexMesh_3x3=False,use_petsc=True,name="_hexMesh_"):
```

Comment:
why doesn't this change/break the existing test for non osx builds?

Reply:
I might have accidentally changed this.  However I'm not sure why it didn't change the outcome of the tests...

Reply:
I think this is not being run currently.